workspace/src/app/views/shared/RuleEditor/view/ruleNode/RuleParameterInput.tsx

```diff
@@ -20,7 +20,7 @@ import { TextAreaWithCharacterWarnings } from "../../../extendedGuiElements/Text
 import { IPropertyAutocomplete } from "@ducks/common/typings";
 import { LanguageFilterProps, PathInputOperator } from "./PathInputOperator";
 import { supportedCodeRuleParameterTypes } from "../../RuleEditor.typings";
-import { SupportedCodeEditorModes } from "@eccenca/gui-elements/src/extensions/codemirror/CodeMirror";
+import { SupportedCodeEditorModes } from "@eccenca/gui-elements/src/extensions/codemirror/hooks/useCodemirrorModeExtension.hooks";
```

Comment:
Always if we need an import that is not available via `@eccenca/gui-elements` base without adding sub paths, then we should think if we do it correct in gui elements.

In this case you could also change it to importing `CodeEditorProps` and use `CodeEditorProps["mode"]`.